dev/bots/prepare_package.dart

```diff
@@ -196,14 +198,14 @@ class ArchiveCreator {
     bool subprocessOutput = true,
     this.platform = const LocalPlatform(),
     HttpReader httpReader,
-  }) : assert(revision.length == 40),
-       flutterRoot = Directory(path.join(tempDir.path, 'flutter')),
-       httpReader = httpReader ?? http.readBytes,
-       _processRunner = ProcessRunner(
-         processManager: processManager,
-         subprocessOutput: subprocessOutput,
-         platform: platform,
-       ) {
+  })  : assert(revision.length == 40),
```

Comment:
I think this is something that @a14n recently fixed.  Plan is to get a plugin availalbe to the analyzer to check this.  But for now I suspect we shouldn't re-introduce the extra space.

Reply:
It's not broken, actually.  That's how dartfmt formats it, and Ian has explicitly said that the extra spaces are fine.

Reply:
Ahh I see - sorry, this isn't one of the cases that his changes impacted.  sorry about that

Reply:
See https://github.com/flutter/flutter/pull/25251#pullrequestreview-184323664

I'm not aware of the final decision about that and I'm interested because at some point I will add a rule to check this style.

Reply:
I don't think it counts as a "final decision", but Ian expressed "not having a strong opinion about this" a while ago: https://github.com/dart-lang/dart_style/issues/658#issuecomment-371346116